src/main/kotlin/no/item/xp/plugin/util/NodeParserHelpers.kt

```diff
@@ -0,0 +1,20 @@
+package no.item.xp.plugin.util
+
+import arrow.core.Option
+import no.item.xp.plugin.extensions.*
+import org.w3c.dom.Node
+
+fun getCommentForGeneratedField(node: Node): Option<String> =
+  node.getChildNodeAtXPath(getXpathExpressionFromString("//label"))
+    .flatMap(Node::getTextContentAsOption)
+
+fun getNameForGeneratedField(node: Node): Option<String> =
+  node.getAttributeAsOption("name")
+
+fun mapToString(node: Node): Sequence<String> {
```

Comment:
I think the name could be a bit more explicit here. 
```suggestion
fun getChildNodesTextContent(node: Node): Sequence<String> {
```